docs/WebTransport.md

```diff
@@ -0,0 +1,284 @@
+# Using WebTransport in Kestrel
+
+Kestrel currently implements most of the WebTransport [draft-02](https://ietf-wg-webtrans.github.io/draft-ietf-webtrans-http3/draft-ietf-webtrans-http3.html) specification, except for datagrams. Datagrams will be implemented at a later date. This document outlines how to use the already implemented functionality.
+
+# Running the sample app
```

Comment:
nit: Add a `#` to this and all following headings so only top level header is a single `#`.

Also add a new line between content and heading.